requirements.txt

```diff
@@ -1,10 +1,6 @@
+-f https://download.pytorch.org/whl/torch_stable.html
 altair==4.1.0
-numpy==1.20.3
 pandas==1.2.4
-torch==1.8.1
+torch==1.8.1+cpu
```

Comment:
Hi @jraza19 , something like `pip3 install torch==1.8.1+cpu torchvision==0.9.1+cpu -f https://download.pytorch.org/whl/torch_stable.html` should work. The cpu version is meant for reduce the app size to optimize deployment.

Reply:
Hi @xudongyang2, i tried the above command and I get the same error. 

```
ERROR: Could not find a version that satisfies the requirement torch==1.8.1+cpu (from versions: 0.1.2, 0.1.2.post1, 0.1.2.post2, 1.4.0, 1.5.0, 1.5.1, 1.6.0, 1.7.0, 1.7.1, 1.8.0, 1.8.1)
ERROR: No matching distribution found for torch==1.8.1+cpu
```

From this [post](https://pytorch.org/get-started/previous-versions/), I feel like the above command only is specific to Windows and for Mac, `pip install torch==1.8.0 torchvision==0.9.1` is sufficient. Might be worth having a windows and mac specific requirement's text?

Reply:
Hi @jraza19 , when deploying on Streamlit share, the process will look for 1 'requiremnts.txt' in the specified github repo. As a result, I believe having 2 'requirements.txt' is not a good idea in this scenario. Also, this 'requiremnts.txt' is for installation the required packages on a server, so we do not really need to worry about installing them on a local machine. The most important reason that I use a cpu version of the torch packages is because they are relatively small in size, which makes deployment possible. Hope this makes sense and sorry for the trouble going through installing them on a Mac OS.

Reply:
Thank you! @xudongyang2 - I understand what you mean now! Makes complete sense :)